samples/peripheral/802154_phy_test/Kconfig

```
@@ -51,7 +51,8 @@ config PTT_HW_VERSION
 	range 0 255
 
 config PTT_ANTENNA_DIVERSITY
-	bool "Enable Antenna Diversity (Not Yet Supported)"
+	depends on !SOC_SERIES_NRF53X
+	bool "Enable Antenna Diversity (Experimental Feature)"
```

```suggestion
	bool "Enable Antenna Diversity [EXPERIMENTAL]"
```

I believe this is the typical way to mark experimental features in Kconfig